### PR TITLE
adds config option to disable standard borg

### PR DESCRIPTION
**Pull request by hornygranny**
@Cheridan

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Copypaste, bad code sorry

**Comment by KorPhaeron** [comment]
In all seriousness this should be done like roundstart races instead of a new config option for every cyborg

File: config/game_options.txt [hunk 1]
```diff
@@ -200,6 +200,9 @@ ALLOW_AI
 ## Uncomment to prevent the secborg module from being chosen
 #DISABLE_SECBORG
 
+### Standardborg ###
+## Uncomment to prevent the standardborg module from being chosen
+#DISABLE_SECBORG
```
Comment (Core0verload) [inline config/game_options.txt]: Secborg again?

**Comment by GunHog** [comment]
...Why?

**Comment by optimumtact** [comment]
Because of the stun prod.

You may as well just remove the stun prod

**Comment by Iamgoofball** [comment]
I'm gonna have to :-1: this for the purposes of "we need standard borg as the fallback for when there literally are no other options"

**Comment by PKPenguin321** [comment]
close this PR and do what Kor said

**Comment by optimumtact** [comment]
Don't thumbs me down @GunHog you meme, I'm just explaining why

**Comment by Zilenan91** [comment]
I think it's a decent idea :+1: 

Nobody ever picks any borg but standard borg now since it's the best for killing. After this it's going to be engiborg, then mediborg, then janiborg, then service borg, and then none will be left.

**Comment by RandomMarine** [comment]
@Zilenan91 I see more mediborgs than anything else what the fuck are you talking about?

**Comment by YakumoChen** [comment]
But nobody actually uses the Standard Borgs except when someone is subverting them.

**Comment by Incomptinence** [comment]
Played standard borg. Don't even use baton until after round ends. 

Apparently I'm an epidemic.

**Comment by KorPhaeron** [comment]
Get someone to do this properly please

**Pull request closed without merging**